src/components/utils/useCheckbox.ts

```diff
@@ -34,6 +35,15 @@ export function useCheckbox({
     }, [indeterminate]);
 
     const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        eventBroker.publish({
+            componentId: 'Checkbox',
+            eventId: 'click',
```

Comment:
This event could be fired not only by user clicks, but also by external updates too. Maybe move it to onClick handler?